hack/check-requisites.sh

```diff
@@ -8,30 +8,36 @@
 
 set -eu
 
+MIN_GO_VERSION=13
+MIN_KUBECTL_VERSION=14
+
+green="\e[32m"
+red="\e[31m"
+reset="\e[39m"
 all_found=true
 
-function check {
+check() {
     local exec_name="$@"
     printf "Checking for $exec_name... "
     if ! command -v $exec_name >/dev/null 2>&1; then
-        printf "missing!"
+        printf "${red}missing${reset}"
```

Comment:
Nit: To me, "missing" sounds a bit more alarming than saying "not found" :smiley:.